app/services/api.rb

```diff
@@ -70,4 +74,9 @@ def self.writable_model?(model)
     @writable ||= writable_models.map { |m| m.name.underscore }
     @writable.include?(model.to_s.underscore.singularize)
   end
+
+  def self.publishable_model?(model)
```

Comment:
I'd rather not have application code that is only used in specs - I try to avoid it. If possible, please move that method to the corresponding spec file.